### PR TITLE
[T4A3][W13-B1] Lim Hui Qi

### PR DESCRIPTION
**Pull request by LuMiN0uSaRc**
Use inheritance for reuse benefits

### CODE REVIEW

**Comment by LuMiN0uSaRc** [comment]
Ready for review

**Comment by okkhoy** [comment]
@LuMiN0uSaRc Shouldn't the team ID be B1? Please correct it!

File: src/seedu/addressbook/data/person/Contact.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package seedu.addressbook.data.person;
+
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Contact.java:2]: Please include a header comment to indicate what this class/method does?

File: src/seedu/addressbook/data/person/Contact.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    
+    public Contact(String value, boolean isPrivate) {
+    	this.value = value;
+    	this.isPrivate = isPrivate;
+    }
+    
+    @Override
+    public boolean equals(Object other) {
+        return other == this // short circuit if same object
+                || (other instanceof Address // instanceof handles nulls
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Contact.java:16]: ouch! you see that you are comparing parent class with one other child class instance? think how you would alleviate this problem?

File: src/seedu/addressbook/data/person/Contact.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
+package seedu.addressbook.data.person;
+
+public abstract class Contact{
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Contact.java:3]: you need a space after `Contact` before `{`

**Comment by okkhoy** [comment]
@LuMiN0uSaRc 
Some comments added. Please ack & close the PR after reading comments.

**Comment by LuMiN0uSaRc** [comment]
Noted with thanks!

**Pull request closed without merging**